blogposts/2021/postgres-version-update.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Postgres version update"
```

Comment:
This blog is about so much more than this and the title should reflect that. The key piece of information here is that we are requiring users to upgrade, and also include a great explanation of how we came to this conclusion. 

Some title suggestions:

`We're updating the minimum supported version of Postgres: Here's why.` 
`Why we're updating the minimum supported version of Postgres`

Reply:
Updated.